hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -439,28 +424,23 @@ public String getCACertificate() throws IOException {
   }
 
   /**
-   *
-   * @param role            - node role: OM/SCM/DN.
-   * @param startSerialId   - start certificate serial id.
-   * @param count           - max number of certificates returned in a batch.
-   * @param isRevoked       - whether list for revoked certs only.
+   * @param role          - node role: OM/SCM/DN.
+   * @param startSerialId - start certificate serial id.
+   * @param count         - max number of certificates returned in a batch.
    * @return
    * @throws IOException
    */
   @Override
   public List<String> listCertificate(NodeType role,
-      long startSerialId, int count, boolean isRevoked) throws IOException {
-    List<X509Certificate> certificates =
-        scmCertificateServer.listCertificate(role, startSerialId, count,
-            isRevoked);
+      long startSerialId, int count) throws IOException {
+    List<X509Certificate> certificates = scmCertificateServer.listCertificate(role, startSerialId, count);
     List<String> results = new ArrayList<>(certificates.size());
     for (X509Certificate cert : certificates) {
       try {
         String certStr = getPEMEncodedString(cert);
         results.add(certStr);
       } catch (SCMSecurityException e) {
-        throw new SCMSecurityException("listCertificate operation failed.",
-            e, e.getErrorCode());
+        throw new SCMSecurityException("listCertificate operation failed.", e, e.getErrorCode());
```

Comment:
NIT: unnecessary change

Reply:
I have changed this line, as there was already a change in this method, so it felt natural to reformat the method body, as this can fit into the new line length limit, and with that there won't be wrapping anymore in the method. I can change this back if you want though.